### PR TITLE
Fix so that fastq files older than three weeks are compressed

### PR DESCRIPTION
**Pull request by moonso**
This PR adds a feature so that only fastq files older than three weeks can be compressed

### Expected test outcome
Only fastq files older than three weeks will be sent for spring compression

## Review
- [x] code approved by MR
- [x] "Merge and deploy" approved by MM
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg/meta/compress/files.py [hunk 1]
```diff
@@ -154,4 +162,12 @@ def check_fastqs(compression_obj: CompressionData) -> bool:
...
         return False
 
+    date_changed = compression_obj.get_change_date(compression_obj.fastq_first)
+    delta: datetime.datetime = date_changed + FASTQ_DATETIME_DELTA
+    today = datetime.datetime.now()
+    # Check if date is older than FASTQ_DELTA
+    if delta > today:
+        LOG.info("FASTQ files are not old enough")
+        return False
+
```
Comment (Mropat) [inline cg/meta/compress/files.py:172]: Could this logic be made more intuitive?
Rephrazing it as:
date changed < today - FASTQ_DATETIME_DELTA
I think is more readable

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,21 +13,26 @@ Try to use the following format:
 ### Changed
 ### Fixed
 
+## [13.17.2]
+
+### Changed
+- Only fastq files older than three weeks will be compressed
+
+
 ## [13.7.1]
```
Comment (Mropat) [inline CHANGELOG.md:22]: can you please fix this typo? :stuck_out_tongue:

**Review by Mropat — APPROVED** [review]
Nice stuff! 
Some minor comments.

File: cg/meta/compress/files.py (hunk 1 above)
Comment (moonso) [reply cg/meta/compress/files.py:172]: Sure

File: CHANGELOG.md (hunk 2 above)
Comment (moonso) [reply CHANGELOG.md:22]: Which one?

File: cg/meta/compress/files.py (hunk 1 above)
Comment (moonso) [reply cg/meta/compress/files.py]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (moonso) [reply CHANGELOG.md]: Changed to `13.17.1`

**Comment by moonso** [comment]
Skärmavbild 2020-11-06 kl  15 34 50

Merged and deployed on hasta

**Pull request merged**